devbot/commands/xkcd.py

```diff
@@ -0,0 +1,81 @@
+""" xkcd comic command. """
+import random
+import discord
+from sqlalchemy import or_
+
+import devbot.database as db
+from devbot.registry import Command
+from devbot.tools import api_requests
+
+
+def embed(title, image_url) -> discord.Embed:
+    result = discord.Embed()
+    result.title = title
+    result.set_image(url=image_url)
+    return result
+
+
+@Command(["xkcd"])
+async def xkcd(message_contents, *_args, **_kwargs) -> (str, discord.Embed):
```

Comment:
Here you give as return type (str, discord.Embed) whereas the returns return str or discord.Embed.